src/connector/src/parser/mod.rs

```diff
@@ -729,7 +730,8 @@ async fn into_chunk_stream<P: ByteStreamSourceParser>(mut parser: P, data_stream
 }
 
 pub trait AccessBuilder {
-    async fn generate_accessor(&mut self, payload: Vec<u8>) -> Result<AccessImpl<'_, '_>>;
+    // TODO(eh)
```

Comment:
```suggestion
    // TODO(error-handling)
```

Reply:
Maybe no one will known what `eh` is later. Also for several other places.

Reply:
Oops, I must have forgotten to clean them up.